cmake/src/hello_world.cpp

```diff
@@ -0,0 +1,7 @@
+#include "../include/hello_world.h"
```

Comment:
Modifie so that you could just write
#include "hello_world.h"

Reply:
++